dapp/src/components/buySell/SellWidget.js

```diff
@@ -183,14 +215,23 @@ const SellWidget = ({
 
     setSellWidgetState('waiting-user')
 
+    const minStableCoinsReceivedBN = ethers.utils.parseUnits(
+      (Math.floor(minStableCoinsReceived * 10000) / 10000).toString(),
+      18
```

Comment:
nit: Alternatively, could do like this: 
```ethers.utils.parseUnits(Math.floor(minStableCoinsReceived * 10000).toString(), 22)```

But I guess it'd make things complex

Reply:
Yup good point. Thought I would rather keep it as it is, since I think it makes the code more readable. I would quickly overlook that 22.